tools/build/configure.py

```diff
@@ -620,16 +620,28 @@ def build(
             "item_data",
         )
 
-        build(
-            [
-                self.build_path() / "include/battle/actor_types.inc.c",
-                self.build_path() / "include/battle/actor_types.h",
-            ],
-            [
-                Path("src/battle/actors.yaml"),
-            ],
-            "actor_types",
-        )
+        if self.version == "jp":
+            build(
+                [
+                    self.build_path() / "include/battle/actor_types.inc.c",
+                    self.build_path() / "include/battle/actor_types.h",
+                ],
+                [
+                    Path("src/battle/actors_jp.yaml"),
+                ],
+                "actor_types",
+            )
+        else:
+            build(
+                [
+                    self.build_path() / "include/battle/actor_types.inc.c",
+                    self.build_path() / "include/battle/actor_types.h",
+                ],
+                [
+                    Path("src/battle/actors.yaml"),
+                ],
+                "actor_types",
+            )
```

Comment:
Had to create a new actors yaml file for JP. Not sure if this is the way we want to handle that or if it's better to somehow support all versions in the same file.

Reply:
I think for now this is fine and we can handle it later when we handle other assets unification between versions